ipad/Utilities/Core/UI/ScrollAlert.swift

```diff
@@ -0,0 +1,95 @@
+//
```

Comment:
Had to create a new class for this new alert message, rather than edit the existing class.